README.md

```diff
@@ -1 +1,4 @@
-# Set-up-a-Hello-Microverse-project
\ No newline at end of file
+Hello Microverse Project
+
+This project includes configuration of linters and also includes gitignore file.
+It follows/use github workflow.
```

Comment:
- [ ] As the previous suggested, It will be nice if you could please extend the readme of your project. You can use the [Microverse Readme Template](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) as an example. Feel free to personalize it.

Reply:
I'm sorry, I don't get it clearly.
Is it the topic or the description you want me to change?
Thank you.

Reply:
What I am trying to point here is, your readme should describe your project. Try giving a brief description like "This is a simple project that shows a styled heading".
You can also list the language or technologies used to build it and the details of the author. 
The template link I shared with you can be used as a guide in coming up with a professional readme.